### PR TITLE
[CI] Install sudo inside containers

### PR DESCRIPTION
**Pull request by alexbatashev**
Additionally make sure pre-commit builds are run for changes like this.

### CODE REVIEW

**Review by bader — DISMISSED** [review]
Ubuntu image doesn't have `sudo`. Wow.

**Comment by bader** [comment]
@alexbatashev, shouldn't we run CI Containers job in pre-commit for changes like this?

**Comment by alexbatashev** [comment]
> @alexbatashev, shouldn't we run CI Containers job in pre-commit for changes like this?

Good point. Fixed.

**Review by bader — APPROVED** [review]

File: .github/workflows/sycl_containers.yaml [hunk 1]
```diff
@@ -10,11 +10,15 @@ on:
     paths:
       - 'devops/containers/**'
       - 'devops/dependencies.json'
+      - 'devops/scripts/install_drivers.sh'
+      - 'devops/scripts/install_build_tools.sh'
```
Comment (bader) [inline .github/workflows/sycl_containers.yaml:14]: Idea for future refactoring. It make sense either move scripts for configuring containers to `devops/containers/**` or come up with some naming scheme, so we can use wildcards to reduce amount of changes in this file.

**Pull request merged**